utils/installer/install.sh

```diff
@@ -167,6 +203,29 @@ function install_nodejs_deps() {
   echo "All NodeJS dependencies are succesfully installed"
 }
 
+function __install_nodejs_deps_yarn() {
+  echo "Installing node modules with yarn.."
+  yarn global add "${__npm_deps[@]}"
+  echo "All NodeJS dependencies are succesfully installed"
+}
+
+function install_nodejs_deps() {
+  local -a pkg_managers=("yarn" "npm")
+  for pkg in "${pkg_managers[@]}"; do
```

Comment:
`for pkg_manager in "${pkg_managers[@]}"`

Reply:
fixed in 2d28c78